firedrake/mesh.py

```diff
@@ -513,8 +520,22 @@ def topological(self):
         return self
 
     def ufl_cell(self):
-        """The UFL :class:`~ufl.classes.Cell` associated with the mesh."""
-        return self._ufl_cell
+        """The UFL :class:`~ufl.classes.Cell` associated with the mesh.
+
+        Note that, by convention, the UFL cells which specifically
```

Comment:
Use
```
.. note:
   
   Indented block
```

For notes.

Reply:
Updated in amended commit 91e48d79b715b47662488044c90a33625d54e63d